composer.json

```diff
@@ -14,7 +14,7 @@
 	"minimum-stability": "dev",
 	"prefer-stable": true,
 	"require": {
-		"php": ">=5.5.0",
+		"php": ">=5.6.0",
```

Comment:
Do we use a feature from PHP 5.6? If no, we should stay with 5.5 until we will require at least 7.0 in the next year

Reply:
The lib `zendframework/zend-diactoros` provides the function `marshalHeadersFromSapi();`.

Some copy pasta:
```php
namespace Zend\Diactoros;

use function array_key_exists;
use function strpos;
use function strtolower;
use function strtr;
use function substr;

/**
 * @param array $server Values obtained from the SAPI (generally `$_SERVER`).
 * @return array Header/value pairs
 */
function marshalHeadersFromSapi(array $server)
{
```
More copy pasta

> Use function and const available in PHP 5.6 only

The 1.3.11 of `zendframework/zend-diactoros` is still PHP 5.4 according to the `composer.json`, but the bundled version has a
```json
"require": {
  "php": "^5.6 || ^7.0",
  "psr/http-message": "^1.0"
},
```
inside.

Reply:
OK, it's hard to build a TER package for PHP 5.5 as our deployment system has PHP 5.6

Reply:
I know what you mean.

I'm so \**censored*\* happy to have kicked out our 4.5 compatibility.
Still, the 5.3 nightmare remains (for now).